best_practices/security.rst

```diff
@@ -173,6 +182,21 @@ the same ``getAuthorEmail`` logic you used above:
 
             return false;
         }
+
+        /**
+         * Checks if the user token has the given role taking into account the
+         * entire role hierarchy defined by the application.
+         */
+        protected function hasRole($roleName, TokenInterface $userToken)
```

Comment:
wouldn't `userHasRole` be a better name. It now looks like a hasser/isser of the class itself